### PR TITLE
RGR-678 Padding on right side of chat bubble is too far from the edge

### PR DESCRIPTION
**Pull request by taj-averylane**
Problem
=======
problem statement, including
JIRA ticket: [RGR-678](https://drivevariant.atlassian.net/browse/RGR-678)

Solution
========
What I/we did to solve this problem
with @pairperson1

Screenshots (optional):
-----------------------
Android | iOS
-------- | --------
Screenshot_1642606042 | Simulator-iPhone12-2022-01-19at09 13 33

### CODE REVIEW

File: src/components/Bubble.tsx [hunk 1]
```diff
@@ -615,15 +615,13 @@ function localStyleSheet(theme: ReactNativePaper.Theme) {
         alignItems: 'flex-start',
       },
       wrapper: {
-        borderRadius: 12,
+        borderRadius: 10,
```
Comment (taj-averylane) [inline src/components/Bubble.tsx:618]: It has `10px` radius on the Figma

File: src/components/Bubble.tsx [hunk 2]
```diff
@@ -645,14 +643,13 @@ function localStyleSheet(theme: ReactNativePaper.Theme) {
...
         backgroundColor: theme.colors.chat.bubbleSent,
         marginLeft: 60,
         minHeight: 20,
         justifyContent: 'flex-end',
-        paddingLeft: 5,
-        paddingRight: 17,
         paddingVertical: 4,
+        borderBottomEndRadius: 10,
```
Comment (taj-averylane) [inline src/components/Bubble.tsx:652]: `borderBottomEndRadius` has the same value with `borderTopEndRadius`(2px) unless specifying it explicitly.

File: src/components/Message.tsx [hunk 3]
```diff
@@ -100,7 +100,8 @@ function localStyleSheet(theme: ReactNativePaper.Theme) {
         lineHeight: Font.LineHeight.extraLarge,
         color: theme.colors.common.white,
         textAlign: 'left',
-        marginBottom: 5,
+        marginTop: 9,
+        marginBottom: 7,
```
Comment (taj-averylane) [inline src/components/Message.tsx:104]: Small tweaking to match with our Figma

**Review by ajp8164 — APPROVED** [review]
LGTM

**Pull request merged**